ServiceLayer/Services.cs

```diff
@@ -0,0 +1,133 @@
+﻿using DAL.Controllers;
+using DAL.Model;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace ServiceLayer
+{
+    public class Services
+    {
+        Response response;
+        public Services()
+        {
+            response = new Response();
+        }
+        public Response GetById(int id, BookRepository bookRepository)
+        {
+            if (id < 0)
```

Comment:
Instead validating everything here, you can create a Validation class as a Service and use it here. That will also follow the SR principle